client/checkout/api/index.js

```diff
@@ -37,10 +37,6 @@ export default class WCPayAPI {
 			options.betas = betas;
 		}
 
-		if ( betas.includes( 'link_beta_2' ) ) {
-			options.apiVersion = '2020-08-27;link_beta=v1';
```

Comment:
Do we have to remove the API version also?

Reply:
I removed it, and it seems that it's not needed anymore. The order process works without it.